src/mocks.ts

```diff
@@ -163,6 +177,20 @@ export function getUid2Cookie() {
   }
 }
 
+export function removeUid2LocalStorage() {
+  localStorage.removeItem(localStorageKeyName);
```

Comment:
this looks exact the same as `removeValue` in UID2LocalStorageManager? Why we couldn't use that instead?

Reply:
removed!